ParabolInc/parabol
# feat(teams-limit): flag an organization when it has exceeded teams limit

Fixes #7379



**How to test:**

- Create a new organization 
- Add `teamLimits` feature flag to the organization `featureFlags` field (Edit rethinkdb table or run `addFeatureFlagToOrg` mutation from #7526)
- Create 3 teams (For simplicity you can adjust numbers in `teamLimitsCheck.ts` https://github.com/ParabolInc/parabol/pull/7517/files#diff-0ebb6687f73dcbeef95db77a23ed552ac97457d45368062605a87c17330b9f9dR7-R10)
- For each team run 3 different meetings with at least 2 attendees, see that **only** after finishing a 3rd meeting for the 3rd team, the organization got flagged with `tierLimitExceededAt` and `scheduledLockAt` values
- You can check flags by calling `r.db('actionDevelopment').table('Organization').get('PUT_ORG_ID_HERE')`
- Create a new team, remove that new team, see flags were **not** removed
- Remove one sticky team (3 meetings, 2 attendees) check that all flags were removed
- Bring back flags (you can edit the database)
- Upgrade organization to PRO, check that all flags were removed from the organization
- Create a new team instead of removed, run 3 meetings with 2 attendee, see that no flags were set on PRO org.
